test/UnitTests/Microsoft.Testing.Extensions.UnitTests/testsbaseline.txt

```diff
@@ -0,0 +1,65 @@
+﻿Microsoft.Testing.Extensions.UnitTests.Microsoft.Testing.Extensions.UnitTests.AppInsightsProviderTests.Platform_CancellationToken_Cancellation_Should_Exit_Gracefully()
```

Comment:
Do we still use baselining? I thought you disabled it, but even if not, feel free to delete these files in other PR.